experiments/run_utility.py

```diff
@@ -141,90 +141,90 @@ def run_dynamic(aggregators, functions, name_base, sample_size=5):
-                print agg + '_' + f + '_' + sk
+            for f, params in functions.items():
+                print(agg + '_' + f + '_' + sk)
                 base_iterations = params[0]
                 small_window_sizes = params[1]
                 big_window_size = params[2]
 
                 for sw in small_window_sizes:
                     row = [f, big_window_size, sw]
-                    print sw, ':',
+                    print(sw, ':',)
 
                     iterations = base_iterations + big_window_size
                     for i in range(sample_size):
                         runtime = exec_runtime(['../bin/' + name_base + '_benchmark_driver', agg, f, str(big_window_size), str(sw), str(iterations), sk])
-                        print runtime,
+                        print(runtime,)
                         sys.stdout.flush()
                         row.append(runtime)
 
-                    print
+                    print()
                     results.writerow(row)
                     results_file.flush()
 
             results_file.close()
 
 def run_shared_half(aggregators, functions, window_sizes, name_base, sample_size=5):
-    for agg, shared_kind in aggregators.iteritems():
+    for agg, shared_kind in aggregators.items():
         for sk in shared_kind:
             results_file = open('results/' + name_base + '_' + agg + '_' + sk + '.csv', 'w')
             results = csv.writer(results_file)
 
-            for f, params in functions.iteritems():
-                print agg + '_' + f + '_' + sk
+            for f, params in functions.items():
+                print(agg + '_' + f + '_' + sk)
                 base_iterations = params
 
                 for w in window_sizes:
                     big_window_size = w 
                     small_window_size = w / 2
 
                     row = [f, big_window_size, small_window_size]
-                    print big_window_size, small_window_size, ':',
+                    print(big_window_size, small_window_size, ':',)
 
                     iterations = base_iterations + big_window_size
                     for i in range(sample_size):
                         runtime = exec_runtime(['../bin/' + name_base + '_benchmark_driver', agg, f, str(big_window_size), str(small_window_size), str(iterations), sk])
-                        print runtime,
+                        print(runtime,)
                         sys.stdout.flush()
                         row.append(runtime)
 
-                    print
+                    print()
                     results.writerow(row)
                     results_file.flush()
 
             results_file.close()
 
 def run_data(aggregators, functions, durations, data_sets, name_base, latency='', sample_size=5):
-    for data_set, data_file in data_sets.iteritems():
+    for data_set, data_file in data_sets.items():
         exp_filename = 'experiments_to_run.txt'
         with open(exp_filename, 'w') as exp_file:
```

Comment:
Should we say `open(..., 'wt')` to clearly indicate opening as text (as opposed to binary)?